### ISSUE TITLE
Should use "fix" for jira mark as bug and feature for Jira mark as other than bug.

### ISSUE THREAD

**Issue by hpe-ykoehler** [bk201-/jira-prepare-commit-msg#603]

**Comment by hpe-ykoehler** [comment]
Right now seems to hard-coded "chore" but it should use info from Jira to be more clever, or allow the developer to use "fix:", "fix!:", "feat:" or "feat!:".

**Comment by bk201-** [comment]
Hi! Do you mean, that hook has to go to JIRA to retrieve the ticket type and insert it into "chore"?

**Comment by hpe-ykoehler** [comment]
@bk201- yes, and also make sure at the same time that the ticket exists and its resolution is "unresolved", etc.

**Comment by bk201-** [comment]
I understood your points, it sounds good, but for some more complex tools. 
To implement this feature needs:
1. Login, Password or any auth things. The user might forget them, the token might expire, etc. 
2. Jira API library. I have to update it, check how it works, excess package, etc. Yep, I can implement only this API call, but anyway, I have to monitor any API changes. 
3. It requires an internet connection for the commit operation which works offline.
4. Each mistake or error will stop the committing process, forcing the developer to go to console logs, read the error, try to find out the cause, fix it, and become angry. 

Yep, if you don't need it, you can avoid using this feature. 

If more users will ask about this feature, I would implement it, but now, I would decline. 

I leave the issue open if you need this feature just add the thumb up, and after 10-20 thumbs I will implement it.
